CMake/External_Caffe.cmake

```diff
@@ -255,6 +255,13 @@ else()
     )
 endif()
 
+if(fletch_BUILD_CUDA_CSTD98)
```

Comment:
I changed it to FORCE_CUDA_CSTD98 in an upcoming commit, as it is here, it's still an advanced flag. It was required to build it in this fashion on my home Fedora 24 PC.